app/concepts/todo/create.rb

```diff
@@ -1,50 +1,45 @@
 class Todo::Create < Trailblazer::Operation
+  contract do
+    property :title, validates: { presence: true }
+  end
 
-  # Logic for creating a TODO
-  #     if `todo_list` is specified
-  #         create the `todo`
-  #         associated with `todo_list`
-  #     else
-  #         create the `todo`
-  #         create a `user`
-  #         create a default `todo_list`
-  #         associate `todo_list` with `user`
-  #         associate `todo` with `todo_list`
   def process(params)
-    # make sure title is not blank
-    return invalid! if params[:todo][:title].blank?
+    begin
+      validate(params[:todo], Todo.new)
```

Comment:
Nice. I see you have attention to detail when it comes to errors and you deal with it as much as possible. :+1:  In this case, we don't want to rescue from `InvalidContract` because as you will eventually learn, we want the behaviour when we're using the operation from the console or from the automated tests.